modules/local/report.nf

```diff
@@ -804,24 +804,57 @@ def table_values(file_path, header_p, seperator, headers=null){
     def converted_data = [:]
     def idx = 0
     def lines_read = false
+    def missing_value = "NoData"
```

Comment:
Instead of "NoData" as missing, can you just do an empty string? In IRIDA Next this would lead to the metadata key being removed (rather then writing a metadata value with "NoData")?

Reply:
I have made the change, but perhaps it should stay as "NoData" so that if there is a field meant to be populated in IRIDANext, they user knows why a value could be missing? 

e.g. its due to a program not writing out a field, rather than the pipeline not identifying the field

Reply:
Thanks. After some discussion, decided to keep "NoData" since that was the existing behaviour.